wls-gui-wahllokalsystem/src/App.vue

```diff
@@ -188,6 +171,7 @@ import { User, UserLocalDevelopment } from "@/types/User";
 
 const userStore = useUserStore();
 const wahlvorstandStore = useWahlvorstandStore();
+const broadcastCronjobService = useBroadcastCronjobService();
```

Comment:
```suggestion
const { startBroadcastMessageInterval, stopBroadcastMessageInterval } = useBroadcastCronjobService();
```
siehe [Vue Doku](https://vuejs.org/guide/reusability/composables.html#return-values)

Reply:
fixed mit 282e4c804632afebfe8abbf8790ff066e6597e05